ssz-rs/src/merkleization/multiproofs/generalized_index.rs

```diff
@@ -0,0 +1,43 @@
+// From: https://users.rust-lang.org/t/logarithm-of-integers/8506/5
+const fn num_bits<T>() -> usize {
+    std::mem::size_of::<T>() * 8
```

Comment:
```suggestion
    core::mem::size_of::<T>() * 8
```
For `no_std` compatible.